pkgs/development/tools/godot/default.nix

```
@@ -7,16 +7,17 @@ let
   options = {
     touch = libXi != null;
     pulseaudio = false;
+    udev = false;
```

Perhaps we could do
```suggestion
    udev = libudev != null;
```
like the `touch` option?

---

That's the more ideal way to do it, but it doesn't quite work. Even with the `libudev` library included in `buildInputs`, the build process throws:

```
platform/x11/libudev-so_wrap.c:96:10: fatal error: libudev.h: No such file or directory
   96 | #include <libudev.h>
```

So just including `libudev` doesn't allow us to actually enable it for the build. I'd imagine the fix for this is an extra line of wiring somewhere, but I don't really know where. Maybe patching in another `env.ParseConfig` (something like `env.ParseConfig("pkg-config libudev --cflags --libs")`) would do it, but that's mostly an uninformed hunch.

---

Looks like my hunch was right. Should build with or without `libudev` now.